packages/salesforcedx-vscode-lwc/src/index.ts

```diff
@@ -73,7 +73,10 @@ export async function activate(context: vscode.ExtensionContext) {
   if (workspaceType === lwcLanguageServer.WorkspaceType.SFDX) {
     await populateEslintSettingIfNecessary(
       context,
-      vscode.workspace.getConfiguration()
+      vscode.workspace.getConfiguration(
+        '',
+        vscode.workspace.workspaceFolders[0].uri
```

Comment:
Usually on Windows we use `uri.fsPath` for the LSP but here we just use `uri` since it's only affecting VS Code and not any LSP.